from clippings.seedwork.exceptions import DomainError


class AuthError(DomainError):
    pass
